project/src/components/property-screen-component/property-screen-component.tsx

```diff
@@ -162,7 +187,7 @@ function PropertyScreen(): JSX.Element | null {
         </div >
       )}
       {
-        !currentOffer && (
+        currentOffer === null && (
           < NotFoundScreen />
```

Comment:
```suggestion
          <NotFoundScreen />
```

как-то странно, что я не заметил, что ты решила так делать. Не надо. Пиши в одном стиле.